### ISSUE TITLE
Is providing a tooltip on hover sufficient for meeting 1.4.1: Use of Color? 

### ISSUE THREAD

**Issue by lr-ninja** [w3c/wcag#3897]
We have colored icons that we use to represent different categories of technical problems. The color we use for a "Needs attention" category does not meet the contrast ratio of 4.5:1. However, these icons have a tooltip on hover that explain what the icons mean. Is this mechanism sufficient?

Test procedure says: Check that the information conveyed is also available in text and that the text is not conditional content.

Does tooltip count as conditional content? Some additional examples would help in the document
https://www.w3.org/WAI/WCAG21/Techniques/general/G14

<img width="83" alt="image" src="https://github.com/w3c/wcag/assets/104940866/fa2ef118-0591-4c21-94df-7a75288e68f5">

<img width="201" alt="image" src="https://github.com/w3c/wcag/assets/104940866/5253dfbc-180e-4d5a-8dd5-90584e96e206">

**Comment by nattarnoff** [comment]
This isn't sufficient. The icon needs to be discernible on its own without the tooltip using shape, border, bolding, contrast, or any other visual cue.. However, as it is a graphic, it only needs to meet 3:1 contrast ratio. 

Don't forget for WCAG 2.2, those target areas need to be 24px by 24px or larger.

**Comment by mraccess77** [comment]
Check out the Dynamic Examples section of SC 1.4.11 https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast which indicates that the control must always have contrast for identification - but other information can be communicated on focus/hover.

**Comment by bruce-usab** [comment]
@lr-ninja if you are content with responses provided, please close this issue.

**Comment by mbgower** [comment]
@lr-ninja As pointed out by @mraccess77, the example you provide is a failure of [1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast). Any operable component must have a contrast of at least 3:1. I'm closing the issue as answered. If you have any follow up questions, feel free to reopen.

**Comment by lr-ninja** [comment]
Appreciate the help and shared links!
